package.json

```diff
@@ -20,6 +20,7 @@
   "homepage": "https://github.com/DKosmos/portfolio#readme",
   "dependencies": {
     "body-parser": "^1.17.2",
-    "express": "^4.15.3"
+    "express": "^4.15.3",
+    "fs": "0.0.1-security"
```

Comment:
fs is a native module, so we don't have to add it over here :) (I think it won't you error even if you have it here though..)